machines/settings.ts

```diff
@@ -2,23 +2,23 @@ import { ContextFrom, EventFrom, send, sendUpdate, StateFrom } from 'xstate';
 import { createModel } from 'xstate/lib/model';
 import { AppServices } from '../shared/GlobalContext';
 import { SETTINGS_STORE_KEY } from '../shared/constants';
-import { VIDLabel } from '../types/vc';
+import { VCLabel } from '../types/vc';
 import { StoreEvents } from './store';
 
 const model = createModel(
   {
     serviceRefs: {} as AppServices,
     name: '',
-    vidLabel: {
+    VCLabel: {
```

Comment:
Can you change this (and other references) to `vcLabel` to distinguish from the `VCLabel` type?